recycling-server/db/migrate/20170302234459_create_categories.rb

```diff
@@ -4,6 +4,12 @@ def change
       t.string :name, null: false
       t.text :description
       t.string :image_link
+      t.string :recycle_class # Five digit string, "abcde"
```

Comment:
We are using five digits actually. So "10000" means recyclable. "10001" means recyclable and hazardous. I was using letters as placeholders. I understand this is more error prone, but it allows for multiple overarching category for each category.